tools/ea/Ea.hs

```diff
@@ -234,10 +234,12 @@ genTxModules = void $ do
     putStrLn "Done."
   where
     gen tag remeds = genTxModule tag $ upgrades <> remeds
-    genOtherTxs = gen "Other" []
-    genDevTxs = gen "Development"
+    genOtherTxs = gen "Other"
       ["pact/coin-contract/remediations/devother/remediations.yaml"]
```

Comment:
The "Other" transactions were empty, meaning the testing Chainweb versions were not seeing upgrades or remediations, and "Development" was using the remediations intended for these other versions (they are identical yamls). So this change will inevitably cause hash breakages in our testing golden tests,